### PR TITLE
Add support for deprecated field

### PR DESCRIPTION
**Pull request by ioanatia**
`deprecated` is one of the fixed fields in an `Operation` object.

It's included in all versions:
- https://github.com/OAI/OpenAPI-Specification/blob/master/versions/1.2.md#523-operation-object
- https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operation-object
- https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#operation-object

### CODE REVIEW

File: lib/grape-swagger/endpoint.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 require 'active_support/core_ext/string/inflections.rb'
 
 module Grape
+  # rubocop:disable Metrics/ClassLength
```
Comment (LeFnord) [inline lib/grape-swagger/endpoint.rb]: please don't add it here, specify it in the .rubocop-todo.yml

**Comment by LeFnord** [comment]
thanks @ioanatia …

File: lib/grape-swagger/endpoint.rb (hunk 1 above)
Comment (ioanatia) [reply lib/grape-swagger/endpoint.rb]: okay, and I will fix the other two rubocop issues that causes the CI to fail.

**Comment by LeFnord** [comment]
thanks again @ioanatia

**Comment by ioanatia** [comment]
thanks for reviewing this so quickly :smile_cat:

**Pull request merged**